app/src/main/java/com/x8bit/bitwarden/ui/platform/components/badge/NotificationBadge.kt

```diff
@@ -0,0 +1,63 @@
+package com.x8bit.bitwarden.ui.platform.components.badge
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Badge
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.x8bit.bitwarden.ui.platform.theme.LocalNonMaterialColors
+
+/**
+ * Reusable component for displaying a notification badge.
+ *
+ * @param notificationCount optional value to display in center of the badge.
+ */
+@Composable
+fun NotificationBadge(
+    modifier: Modifier = Modifier,
+    notificationCount: Int = 0,
```

Comment:
Can we drop the default on `notificationCount`, I think we want that passed in every time.

Reply:
Sure!